toolchain/check/type_completion.cpp

```diff
@@ -123,7 +123,7 @@ class TypeCompleter {
   template <typename InstT>
     requires(InstT::Kind.template IsAnyOf<
              SemIR::AssociatedEntityType, SemIR::CppOverloadSetType,
-             SemIR::CustomCppVoidType, SemIR::FunctionType,
+             SemIR::CppVoidType, SemIR::FunctionType,
```

Comment:
Sorry, I'd missed this before... Can you please break `CppVoidType` out to its own function, and add a TODO that it should be always-incomplete?

That is, the changes in this file have the consequence of always treating `void` as a complete type. However, functionally we don't want it to behave that way -- only pointers are valid. So it should probably never be set as complete.

That should make `todo_fail_void.carbon` fail, for example.